ddtrace/internal/writer.py

```diff
@@ -47,6 +48,29 @@
 # to 10 buckets of 1s duration.
 DEFAULT_SMA_WINDOW = 10
 
+DEFAULT_BUFFER_SIZE = 8 * 1000000  # 8mb
+DEFAULT_MAX_PAYLOAD_SIZE = 8 * 1000000  # 8mb
+DEFAULT_PROCESSING_INTERVAL = 1.0
+
+
+def get_writer_buffer_size():
+    # type: () -> int
+    return int(get_env("trace", "writer_buffer_size_bytes", default=DEFAULT_BUFFER_SIZE))  # type: ignore[arg-type]
```

Comment:
Since there never was a `DATADOG_` prefixed env var for this, does it still make sense to use `get_env` in this case? If so, then maybe do
```suggestion
    return int(get_env("trace", "writer", "buffer_size_bytes", default=DEFAULT_BUFFER_SIZE))  # type: ignore[arg-type]
```
?

Reply:
using `get_env` is nice at least to handle some common behavior, or if we decide to add other common behavior to env var extraction later, but your point does make sense!

I had issues with the line length and flake8 mostly how I ended up with what we have, removing the extra `", "` helps, and shouldn't matter much either way since we just do `"_".join(args)` anyways.

no strong opinion if you want me to change it, let me know!

Reply:
It's ok, it was just a suggestion, no strong opinion from my side as well. I just thought that if we want to go with `env_var` it made sense to have those finer-grained "breadcrumbs", given the way `env_var` works. But I'm also fine the way it is now.